ntbs-service/wwwroot/source/Components/InactivityChecker.ts

```diff
@@ -0,0 +1,50 @@
+﻿import Vue from "vue";
+import InactivityLeaveComponent from "./InactivityLeaveComponent";
+import { getHeaders, buildPathRelativeToOrigin } from "../helpers";
+import axios from "axios"; 
+
+const InactivityChecker = Vue.extend({
+    render(): any { return null },
+    mounted: function() {
+        this.interval = setInterval(this.checkActivity, 60000);
+    },
+    
+    methods: {
+        checkLeave: function () {
+            this.$modal.confirm(InactivityLeaveComponent, "You have been inactive")
+                .then(() => {
+                    window.location.href = buildPathRelativeToOrigin("Logout")
+                })
+                .catch(() => {
+                    this.updateActivity();
+                    this.$modal.close()
+                });
+        },
+        checkActivity() {
```

Comment:
Small thing - I would put this as the first method in the list, so the code reads more in order :)